SPECS/flux/flux.spec

```diff
@@ -0,0 +1,178 @@
+#
+# spec file for package flux
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+%global rustflags '-Clink-arg=-Wl,-z,relro,-z,now'
+
+%define libflux_suffix %(echo %{version} | tr . _)
+
+Summary:        Influx data language
+Name:           flux
+Version:        0.179.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Productivity/Databases/Servers
+URL:            https://github.com/influxdata/flux
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# Note: the %%{name}-%%{version}-cargo.tar.gz file contains a cache created by capturing the contents downloaded into $CARGO_HOME.
+# To update the cache and config.toml run:
+#   tar -xf %{name}-%{version}.tar.gz
+#   cd %{name}-%{version}/libflux
+#   cargo vendor > config.toml
+#   tar -czf %{name}-%{version}-cargo.tar.gz vendor/
+#
```

Comment:
For the sake of automatic updates, I think we're switching to using scripts checked into the repo instead of in-spec comments. See `SPECS/etcd/generate_source_tarball.sh` as an example. Having said that, for cargo tarballs we've been using `toolkit/scripts/build_cargo_cache.sh` for now (example in `rust.spec`). Please check with @nicogbg what is his preference here.